Addition issues with /tmp

Addition issues with regards to tmp file solution (#23 #24).

Unable to provide a MWE at this time.

I have a slight suspicion that some plugin use the location of the pom file to determine the project directory. So, the solution #23 of using a tmp file for the inlined POM causes problem.

Might have to re-track, place the inlined-pom in the same directory as the pom.xml but remove the inlined-pom at the end of the execution.

Using 0.3.2:

```
$ cat .mvn/extensions.xml || true
<?xml version="1.0" encoding="UTF-8"?>
<extensions>
    <extension>
        <groupId>eu.maveniverse.maven.nisse</groupId>
        <artifactId>extension3</artifactId>
        <version>0.3.2</version>
    </extension>
</extensions>
```

Different log outpus as follows.


1. resources plugin loogs for files in tmp dir
```
[INFO] --- resources:3.3.1:resources (default-resources) @ ms001 ---
[INFO] skip non existing resourceDirectory /tmp/src/main/resources
[INFO] Copying 2 resources from ../../../../../tmp/target/lips3-common-resources to ../../../../../tmp/target/classes

...

[INFO] --- resources:3.3.1:testResources (default-testResources) @ ms001 ---
[INFO] skip non existing resourceDirectory /tmp/src/test/resources
[INFO] skip non existing resourceDirectory /tmp/src/integration-test/resources
[INFO] skip non existing resourceDirectory /tmp/src/microservice-test/resources
```

2. compiler uses tmp dir

```
[INFO] --- compiler:3.13.0:testCompile (default-testCompile) @ ms001 ---
[INFO] Recompiling the module because of changed source code.
[INFO] Compiling 1 source file with javac [debug release 21] to ../../../../../tmp/target/test-classes
```

3. surefire uses tmp dir

```
[INFO] Running se.lfv.lips3.microservices.ms001.SpringBootAppIT
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.419 s <<< FAILURE! -- in se.lfv.lips3.microservices.ms001.SpringBootAppIT
[ERROR] se.lfv.lips3.microservices.ms001.SpringBootAppIT -- Time elapsed: 1.419 s <<< ERROR!
java.lang.IllegalStateException: Unable to find a @SpringBootConfiguration, you need to use @ContextConfiguration or @SpringBootTest(classes=...) with your test
	at org.springframework.util.Assert.state(Assert.java:76)
...

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project ms001: 
[ERROR] 
[ERROR] Please refer to /tmp/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project ms001: 
Please refer to /tmp/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.

```

--- a/extension3/src/main/java/eu/maveniverse/maven/nisse/extension3/internal/NissePropertyInliner.java
+++ b/extension3/src/main/java/eu/maveniverse/maven/nisse/extension3/internal/NissePropertyInliner.java
@@ -64,8 +64,7 @@ final class NissePropertyInliner {
                 if (isAnyKeyPresent(inlinedProperties, pomPath)) {
                     // needs rewrite
                     logger.info(" * {}:{} needs inlining", mavenProject.getGroupId(), mavenProject.getArtifactId());
-                    Path inlinedPomPath = Files.createTempFile(".inlined", ".xml");
-                    Files.createDirectories(inlinedPomPath.getParent());
+                    Path inlinedPomPath = pomPath.getParent().resolve(".inlined-" + pomPath.getFileName());
                     session.getRepositorySession().getData().set(INLINED_POM_PATH_KEY, inlinedPomPath);
                     inline(pomPath, inlinedPomPath, inlinedProperties);
                     mavenProject.setPomFile(inlinedPomPath.toFile());

